upgrade_versions.py

```diff
@@ -212,18 +213,49 @@ def _read_version_from_repo(self, repo_path):
             raise exception.PackageError(msg)
 
 
+def push_new_versions(versions_repo, release_date, versions_repo_push_url,
+        versions_repo_push_branch, committer_name, committer_email):
+    """
+    Push updated versions to the remote Git repository, using the
+    system's configured git committer and SSH credentials.
+    """
+    LOG.info("Pushing packages versions updates on release dated {date}"
+             .format(date=release_date))
+
+    LOG.info("Creating remote for URL {}".format(versions_repo_push_url))
+    VERSIONS_REPO_REMOTE = "push-remote"
+    versions_repo.create_remote(VERSIONS_REPO_REMOTE, versions_repo_push_url)
+
+    LOG.info("Adding files to repository index")
+    versions_repo.index.add(["*"])
+
+    LOG.info("Committing changes to local repository")
+    commit_message = "Weekly build {date}".format(date=release_date)
+    actor = git.Actor(committer_name, committer_email)
+    versions_repo.index.commit(commit_message, author=actor, committer=actor)
+
+    LOG.info("Pushing changes to remote repository")
+    remote = versions_repo.remote(VERSIONS_REPO_REMOTE)
+    refspec = "HEAD:refs/heads/{}".format(versions_repo_push_branch)
```

Comment:
The workspace of our build machine is volatile and can be deleted after builds.
This pushes the changes to somewhere accessible (the bot's github fork, for example) to ease a possible pull request and merge into the main repository.